src/importlinter/contracts/_common.py

```diff
@@ -137,20 +139,43 @@ def _pop_shortest_chains(graph: ImportGraph, importer: str, imported: str):
             yield chain
 
 
-def format_line_numbers(line_numbers: Sequence[Optional[int]]) -> str:
+def format_line_numbers(format: str, line_numbers: Sequence[Optional[int]]) -> str:
     """
     Return a human-readable string of the supplied line numbers.
 
     Unknown line numbers should be provided as a None value in the sequence. E.g.
     (None,) will be returned as "l.?".
     """
-    return ", ".join(
-        "l.?" if line_number is None else f"l.{line_number}" for line_number in line_numbers
-    )
+    if format == "default":
```

Comment:
I don't think 'default' is the right name for this. It is the default, but maybe something like 'module_name' versus 'file_name' - or even 'module' versus 'file' would make more sense.